lib/apt/build.sh

```diff
@@ -79,7 +79,11 @@ sed -i'' -E "s/^($PACKAGE \(\S+\)) unstable;/\1 $DISTRIBUTION;/g" debian/changel
 if [ "${DEBUG:-no}" = "yes" ]; then
   run debuild -us -uc --lintian-opts --profile td-agent/${distribution}
 else
-  run debuild -us -uc --lintian-opts --profile td-agent/${distribution} > /dev/null
+  if [ "${LINTIAN:-yes}" = "yes" ]; then
```

Comment:
I think lintian check should be done by default, skip it only when a user disables it.

Reply:
"user disables it" means setting LINTIAN=no environment variable by manual,  am I right?

Reply:
Right.

Reply:
Thanks, I got it.